tests/unit/test_beaker_transformer.py

```diff
@@ -114,67 +117,97 @@ async def create_transformer(self, legacy=False):
+                    "whiteboard": default_whiteboard,
+                    "priority": default_prio,
+                    "tasks": [
+                        {"name": "/distribution/check-install", "role": "SERVER"}
+                    ],
+                    "retention_tag": default_retention_tag,
+                    "product": default_product,
+                },
             ),
             (
                 centos,
-                "CentOS-Stream-9%",
-                "BaseOS",
-                "PROV_CONF_CENTOS_KS_META",
-                ["%post\ncat /etc/redhat-release\n%end"],
-                default_whiteboard,
-                default_prio,
+                {
+                    "distro": "CentOS-Stream-9%",
+                    "variant": "BaseOS",
+                    "ks_meta": "PROV_CONF_CENTOS_KS_META",
+                    "ks_append": ["%post\ncat /etc/redhat-release\n%end"],
+                    "whiteboard": default_whiteboard,
+                    "priority": default_prio,
+                    "tasks": default_tasks,
+                    "retention_tag": default_retention_tag,
+                    "product": default_product,
+                },
             ),
             # default variant should be there,
             # windows distro does not exist so host['os'] should be copied
             (
                 windows,
-                "win-2022",
-                "BaseOS",
-                "PROV_CONF_DEFAULT",
-                default_ks_append,
-                "BEAKER DOES NOT SUPPORT WINDOWS THIS JOB MUST FAIL",
-                "ULTRAHIGH",
+                {
+                    "distro": "win-2022",
+                    "variant": "BaseOS",
+                    "ks_meta": "PROV_CONF_DEFAULT",
+                    "ks_append": default_ks_append,
+                    "whiteboard": "BEAKER DOES NOT SUPPORT WINDOWS THIS JOB MUST FAIL",
+                    "priority": "ULTRAHIGH",
+                    "tasks": default_tasks,
+                    "retention_tag": default_retention_tag,
+                    "product": default_product,
+                },
             ),
             (
                 rhel86,
-                "RHEL-8.6%",
-                "BaseOS",
-                "PROV_CONF_RHEL86_KS_META",
-                ["%post\ncat /etc/redhat-release\nwget redhat.com\n%end"],
-                default_whiteboard,
-                default_prio,
+                {
+                    "distro": "RHEL-8.6%",
+                    "variant": "BaseOS",
+                    "ks_meta": "PROV_CONF_RHEL86_KS_META",
+                    "ks_append": [
+                        "%post\ncat /etc/redhat-release\nwget redhat.com\n%end"
+                    ],
+                    "whiteboard": default_whiteboard,
+                    "priority": default_prio,
+                    "tasks": default_tasks,
+                    "retention_tag": default_retention_tag,
+                    "product": default_product,
```

Comment:
Do you think you can add some other than default values to one or two hosts ? 
I mean:
```
retention_tag
product
```
there I can see only defaults.

Reply:
Thanks for the additions. Looks fine now.